### DISCUSSION TITLE
Welcome to Azure Tools for Java

### DISCUSSION THREAD

**Discussion by yucwan** [microsoft/azure-tools-for-java#5087] [General]
# 👋 Hey there!
We’re using Discussions as a place to connect with other Java developers, customers, and users of the [Azure Toolkit for IntelliJ](https://plugins.jetbrains.com/plugin/8053-azure-toolkit-for-intellij), [Azure Toolkit for Eclipse](https://marketplace.eclipse.org/content/azure-toolkit-eclipse) or any other Azure tools for Java.
 
We hope that you:

* Provide feedback on our tools.
* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded.

**Comment by prshnt** [comment]
Are there any plans to add Kusto support on the plugin?
